src/call/mod.rs

```diff
@@ -31,9 +31,10 @@ use crate::codec::{DeserializeFn, Marshaller, SerializeFn};
 use crate::error::{Error, Result};
 use crate::task::{self, BatchFuture, BatchType, CallTag, SpinLock};
 
-pub use crate::grpc_sys::GrpcStatusCode as RpcStatusCode;
+pub use crate::grpc_sys::grpc_status_code as RpcStatusCode;
+pub type RpcStatusCodeType = i32;
```

Comment:
Why not use `RpcStatusCode` directly?

Reply:
In this case, `RpcStatusCode` is a module name, bindgen generates those const in that module. To represent the type of the code, we need a code-type-name.